bin/space2underscore

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env ruby
-
 require 'space2underscore'
 
-result  = Space2underscore.convert(ARGV)
-puts Space2underscore.create_new_branch(result)
+args = ARGV.reject { |arg| /-c|--create/ === arg }
+result  = Space2underscore.convert(args)
+
+if ARGV.include?('-c') || ARGV.include?('--create')
```

Comment:
If you put `%w(-c --create)` in a variable when using it before, you can just do

```
CREATE_FLAGS = %w(-c --create)
args = ARGV.reject { |arg| CREATE_FLAGS.include?(arg) }


if (ARGV & CREATE_FLAGS).empty?
 # do not create
else
  # create
end
```